client/flutter/lib/pages/main_pages/check_up_page.dart

```diff
@@ -1,14 +1,342 @@
 import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:intl/intl.dart';
+import 'package:who_app/api/symptoms.dart';
+import 'package:who_app/components/loading_indicator.dart';
+import 'package:who_app/components/page_button.dart';
+import 'package:who_app/components/page_scaffold/page_header.dart';
 import 'package:who_app/components/page_scaffold/page_scaffold.dart';
+import 'package:who_app/components/themed_text.dart';
+import 'package:who_app/constants.dart';
+
+class CheckUpPage extends StatefulWidget {
+  @override
+  _CheckUpPageState createState() => _CheckUpPageState();
+}
+
+class _CheckUpPageState extends State<CheckUpPage> {
+  List<SymptomResult> timeseries;
+
+  _CheckUpPageState() : super() {
+    SymptomTimeseries().addListener(refreshDashboard);
```

Comment:
IMO it's more intuitive to update data within the `ChangeNotifier`, call `notifyListeners()` then access `FooChangeNotifier.data` via `.addListener` and call setState. 

A read shouldn't be necessary after every write.

Or even better, https://www.youtube.com/watch?v=s-ZG-jS5QHQ, use ValueListenableBuilder
docs: https://api.flutter.dev/flutter/widgets/ValueListenableBuilder-class.html